src/Admin/Support.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Class SupportMenu
+ *
+ * @package Ampproject\Ampwp
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\QueryVar;
+
+/**
+ * SupportMenu class.
+ */
+class Support implements Service, Conditional, Registerable {
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+
+		return current_user_can( 'manage_options' );
+	}
+
+	/**
+	 * Adds hooks.
+	 */
+	public function register() {
+
+		/**
+		 * Add support link to Admin Bar.
+		 */
+		add_action( 'admin_bar_menu', [ $this, 'admin_bar_menu' ], 105 );
+
+		/**
+		 * Add support link to meta box.
+		 */
+		add_filter( 'amp_validated_url_status_actions', [ $this, 'amp_validated_url_status_actions' ], 10, 2 );
+
+		/**
+		 * Add support link to Post row actions.
+		 */
+		add_filter( 'post_row_actions', [ $this, 'post_row_actions' ], PHP_INT_MAX, 2 );
+
+		/**
+		 * Plugin row Support link.
+		 */
+		add_filter( 'plugin_row_meta', [ $this, 'plugin_row_meta' ], 10, 2 );
+
+	}
+
+	/**
+	 * Add Diagnostic link to Admin Bar.
+	 *
+	 * @param \WP_Admin_Bar $wp_admin_bar WP_Admin_Bar object.
+	 *
+	 * @return void
+	 */
+	public function admin_bar_menu( $wp_admin_bar ) {
+
+		if ( empty( $wp_admin_bar ) ||
+			! is_a( $wp_admin_bar, 'WP_Admin_Bar' ) ||
+			! array_key_exists( 'amp', $wp_admin_bar->get_nodes() )
+		) {
+			return;
+		}
+
+		// Get the AMP Validated URL post ID.
+		$current_url = remove_query_arg(
+			array_merge( wp_removable_query_args(), [ QueryVar::NOAMP ] ),
+			amp_get_current_url()
+		);
+
+		$post = \AMP_Validated_URL_Post_Type::get_invalid_url_post( $current_url );
+
+		$wp_admin_bar->add_node(
+			[
+				'parent' => 'amp',
+				'title'  => __( 'Support', 'amp' ),
```

Comment:
```suggestion
				'title'  => esc_html__( 'Support', 'amp' ),
```

Reply:
This says "Support" whereas the others say "Get support" or "Contact support". Should they be consistent?